.github/workflows/ci.yml

```diff
@@ -0,0 +1,54 @@
+name: CI
+
+on:
+  pull_request:
+  push:
+    branches:
+      - main
+
+jobs:
```

Comment:
There's opportunity to DRY this up and only install once / cache `node_modules`, but this is a good start and gets us off of circle.